client-app/src/Components/Login.js

```diff
@@ -1,48 +1,32 @@
 import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
+import '../css/Login.css'; // Import the CSS file
 
-const Login = props => {
+const Login = () => {
     const [credentials, setCredentials] = useState({ email: '', password: '' });
     const [captcha, setCaptcha] = useState('');
     const [captchaValue, setCaptchaValue] = useState('');
     const [errorMessage, setErrorMessage] = useState('');
 
-    localStorage.setItem('isLogged', false);
-
     const generateCaptcha = () => {
         const randomCaptcha = Math.random().toString(36).substring(7);
         setCaptcha(randomCaptcha);
     };
 
-    const handleSubmit = async e => {
+    const handleSubmit = e => {
         e.preventDefault();
 
         // Validate CAPTCHA
         if (captchaValue.toLowerCase() !== captcha.toLowerCase()) {
             setErrorMessage('Incorrect CAPTCHA. Please try again.');
             return;
         }
-        // const response = await fetch("http://localhost:5000/api/auth/login", {
-        //     method: 'POST',
-        //     headers: {
-        //         'Content-Type': 'application/json'
-        //     },
-        //     body: JSON.stringify({ email: credentials.email, password: credentials.password })
-        // });
-        // const json = await response.json();
-
-        // if (json.success) {
-        //     localStorage.setItem('token', json.authtoken);
-        //     localStorage.setItem('name', json.name);
```

Comment:
I would suggest to revert above deletions as they look meaningful and we might refactor it in future